### DISCUSSION TITLE
is there a way to use smartrelative numbers

### DISCUSSION THREAD

**Discussion by victororlyk** [JetBrains/ideavim#934] [Q&A]
in vs code there quite cool feature smart relative numbers, which in normal mode show relative numbers and in input mode absolute. Is there anything similar?

**Comment by citizenmatt** [comment]
There's no support for something like this right now, sorry. At some point, we'd like to support `autocmd`, which would allow running vimscript in response to certain events, such as changing mode. This would likely allow us to `set nornu` when entering Insert mode and `set rnu` when exiting. Here's the ticket for `autocmd`, but unfortunately, there's currently no ETA: [VIM-1693](https://youtrack.jetbrains.com/issue/VIM-1693/Add-support-for-autocmd)
